creator-node/test/prometheus.test.js

```diff
@@ -40,24 +40,63 @@ describe('test Prometheus metrics', async function () {
   })
 
   it('Checks the middleware tracks routes with route params', async function () {
-    app.get('serviceRegistry').prometheusRegistry.regexes = [
-      {
-        regex: /(?:^\/ipfs\/(?:([^/]+?))\/?$|^\/content\/(?:([^/]+?))\/?$)/i,
-        path: '/ipfs/#CID'
-      }
-    ]
-
     await request(app).get('/ipfs/QmVickyWasHere')
     await request(app).get('/content/QmVickyWasHere')
 
     const resp = await request(app).get('/prometheus_metrics').expect(200)
 
```

Comment:
a test to hit an unregistered route and then assert that prometheus metrics response does not include it would be valuable here

Reply:
added test ~